.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+/.github/settings.yml @detiber @displague @micahhausler
+/.github/CODEOWNERS @detiber @displague @micahhausler
```

Comment:
not sure if the missing newline at the EOLmatters, but github is marking it so might as well ensure its in